### ISSUE TITLE
Clear button overrides content

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-combo-box#406]
### Description
Since [this commit](https://github.com/vaadin/vaadin-combo-box/commit/528ca8679dc043aca95e8e0439640bcff9e44e72), clear button is shown over the text which results in the ugly appearance if the value is long enough (or combo box is narrow):

![2017-02-24 16 31 41](https://cloud.githubusercontent.com/assets/10589913/23307093/eb30840a-faae-11e6-8d12-4e71d08c6afc.png)

### Expected outcome
There should be an option (maybe CSS custom property) to override clear icon background color for that. Of course we could always pass a custom clear icon, but this is not the good idea IMO.

### Actual outcome
Not-so-readable combo box with transparent clear button shown over the selected value.

**Comment by web-padawan** [comment]
Added a JSBin: https://jsbin.com/lidera/edit?html,output

**Comment by web-padawan** [comment]
Just realized that setting background is not the good idea to fix that. What about changing thus line:
```
      :host([opened][has-value]) #input {
        padding-right: 60px;
        margin-right: -32px;
      }
```

to just 

```

        padding-right: 60px;
        margin-right: -32px;
      }
```

**Comment by Saulis** [comment]
Released a fix in `v1.3.2` https://github.com/vaadin/vaadin-combo-box/releases
